examples/crm/src/contacts/ContactList.tsx

```diff
@@ -87,24 +81,48 @@ const ContactListActions = () => (
     </TopToolbar>
 );
 
-const exporter: Exporter<Contact> = async (records, fetchRelatedRecords) => {
+const exporter: Exporter<Contact> = async (
+    records,
+    fetchRelatedRecords,
+    dataProvider
+) => {
     const companies = await fetchRelatedRecords<Company>(
         records,
         'company_id',
         'companies'
     );
     const sales = await fetchRelatedRecords<Sale>(records, 'sales_id', 'sales');
-    const tags = await fetchRelatedRecords<Tag>(records, 'tags', 'tags');
```

Comment:
I'm surprised by this change: `fetchRelatedRecords` does a `getMany` and should work. Why replace it with a manual `getList`?

Reply:
This is caused by a bug in ra-data-postgrest's `getMany` implementation. When called with an empty array, it generates an incorrect URL (`http://127.0.0.1:54321/rest/v1/tags?id=eq.`) and Postgrest returns a 400 bad request. Until this is fixed we'll have to live with this code.